ydb/docs/en/core/changelog-cli.md

```diff
@@ -1,5 +1,26 @@
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.18.0 {#2-18-0}
+
+Released on December 24, 2024. To update to version **2.18.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+* Query plan and statistics improvements:
+  * Expression and attributes added to various operator properties (i.e. GroupBy)
+  * Per operator statistics (Rows)
+  * Statistics from Column Shards (Rows and Bytes)
+* Fixed a bug where `ydb workload * run` command could crash in `--dry-run` mode.
+* Added support for views in local backups: `ydb tools dump` and `ydb tools restore`. Views are backed up as `CREATE VIEW` queries saved in the `create_view.sql` files, which can be executed to recreate the original views.
+* Replaced option `--query-settings` by `--query-prefix` one in `ydb workload <workload> run`.
```

Comment:
This needs to be marked as a backward-incompatible change if it doesn't support the old name anymore.